packages/craftcms-vue/admintable/App.vue

```diff
@@ -717,6 +717,10 @@
           }
           this.reload();
         }
+
+        const url = new URL(window.location.href);
+        url.searchParams.set('q', this.searchTerm);
```

Comment:
@brandonkelly I don't think using `q` here will cause any issues, but if you think it's likely to run into another query param, we can use something else.